src/components/Timeline/index.js

```diff
@@ -463,6 +463,7 @@ class Timeline extends Component {
         const style = {
           left: `${(event.route_offset_millis / route.duration) * 100}%`,
           width: `${((event.data.end_route_offset_millis - event.route_offset_millis) / route.duration) * 100}%`,
+          minWidth: '1px',
```

Comment:
2 or 3px too big?

Reply:
they start to overlap with events next to them and some small ones disappear, even on 2px